src/main/java/lesson4/ApacheWorker.java

```diff
@@ -0,0 +1,66 @@
+package lesson4;
+
+import org.apache.http.*;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.impl.client.HttpClientBuilder;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ApacheWorker {
+
+    public static List<String> getListOfRepoUsers() throws Exception {
+
+        String owner = "namellez";
```

Comment:
there are some constants like owner, url, repo that should be declared as constants but they don't.

Reply:
Declared owner and repo strings as constant and moved to class fields
Declared url and user strings as final and left them in methods (they are different for each method)